managementAgent.js

```diff
@@ -0,0 +1,11 @@
+const logger = require('./lib/utilities/logger');
+const { initManagement } = require('./lib/management');
+const _config = require('./lib/Config').config;
+
+process.env.REPORT_TOKEN = process.env.REPORT_TOKEN
+                           || _config.reportToken
+                           || 'management_agent_report_token';
```

Comment:
Something random (like a uuid) would be safer if the deployment system hasn't passed a token explicitly.

Reply:
done